### DISCUSSION TITLE
Manually run subgen on a local folder

### DISCUSSION THREAD

**Discussion by bigdoug2005** [McCloudS/subgen#60] [General]
Is it possible to point subgen at a local folder on Windows and have it generate .srt files for everything in the folder? I am just looking for a simple one-time manually triggered transcription instead of setting up a full running service hooked into Emby, Plex, etc.

**Comment by McCloudS** [comment]
Yes, look at the environment variables for transcribe folders.  You can also use the /batch endpoint.
